### ISSUE TITLE
11th generation Intel Support

### ISSUE THREAD

**Issue by matbb** [rr-debugger/rr#2811]
When running rr on 11th gen Intel I get the following error:
```
[FATAL<my build path>/rr-debugger/src/PerfCounters_x86.h:100:compute_cpu_microarch()] Intel CPU type 0x806c0 unknown
```
Build from source (master at 00413940).
Changing `src/PerfCounters_x86.h` with
```
     case 0x706e0:
+    case 0x806c0:
       return IntelIcelake;
```
makes the error disappear and rr seems to work fine.

I am aware that the architecture is not exactly the same.
Should I expect some problems, or is using an older architecture ok?
Is there another way to make it work ?

My cpu:
```
$ lscpu | grep 'Model name'
Model name:                      11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz
```

**Comment by khuey** [comment]
We should probably add a new `IntelTigerlake` enum variant for it.

Does `make check` pass on your system after you made that modification?

**Comment by matbb** [comment]
Yes, `make check` passes:
```
...
2457: Test 'record_replay_32' PASSED
2553/2553 Test #2457: record_replay-32-no-syscallbuf ....................................   Passed  132.67 sec

100% tests passed, 0 tests failed out of 2553

Total Test time (real) = 604.66 sec
Built target check
build $  
```

**Comment by khuey** [comment]
Great. We would take a PR that looks like b9a17e6d50c2bee08dbcdc3841b44a219e4d603f, though change `LastIntel` while you're in there too. Since treating Tiger Lake as Ice Lake works you can just copy the `pmu_config` values for `IntelIcelake` into a new row for `IntelTigerlake`

**Comment by khuey** [comment]
Closing as fixed by #2812. Thanks!
